samples/net/openthread/ncp/prj-rcp.conf

```diff
@@ -0,0 +1,31 @@
+CONFIG_NETWORKING=y
+CONFIG_NET_L2_OPENTHREAD=y
+
+CONFIG_LOG=y
+
+# Set OpenThread RCP architecture
+CONFIG_OPENTHREAD_COPROCESSOR=y
+CONFIG_OPENTHREAD_COPROCESSOR_RCP=y
```

Comment:
I think it would be cleaner to provide an `overlay-rcp.conf` with the following change, instead of duplicating the entire conf file:
```
CONFIG_OPENTHREAD_COPROCESSOR_NCP=n
CONFIG_OPENTHREAD_COPROCESSOR_RCP=y
```

Reply:
Done. Also the documentation has been adapted.